src/hooks/useBUSDPrice.ts

```diff
@@ -0,0 +1,78 @@
+import { ChainId, Currency, currencyEquals, JSBI, Price, WETH } from '@pancakeswap/sdk'
+import { useMemo } from 'react'
+import useActiveWeb3React from 'hooks/useActiveWeb3React'
+import { BUSD, CAKE } from '../config/constants/tokens'
+import { PairState, usePairs } from './usePairs'
+import { wrappedCurrency } from '../utils/wrappedCurrency'
+
+const BUSD_MAINNET = BUSD[ChainId.MAINNET]
+
+/**
+ * Returns the price in BUSD of the input currency
+ * @param currency currency to compute the BUSD price of
+ */
+export default function useBUSDPrice(currency?: Currency): Price | undefined {
+  const { chainId } = useActiveWeb3React()
+  const wrapped = wrappedCurrency(currency, chainId)
+  const tokenPairs: [Currency | undefined, Currency | undefined][] = useMemo(
+    () => [
+      [
+        chainId && wrapped && currencyEquals(WETH[chainId], wrapped) ? undefined : currency,
+        chainId ? WETH[chainId] : undefined,
```

Comment:
Is there a plan to refactor the continued references to tokens `WETH` / `ETHER` coming from the SDK to reflect BNB/WBNB, and then continue this into the FE so that variables like `ethPairState `, `showETH` etc also refer to the correct ticker?